.github/actions/build-py/action.yml

```diff
@@ -30,8 +30,16 @@ runs:
           cp dist/*.whl wheelhouse
         fi
         cp -r wheelhouse ..
-    - name: Upload Python Wheels
+    - name: Upload Python Wheels Simulation # See Issue #939
```

Comment:
build-py fails for simulation with the newer action versions, so this needs to be setup in a different way. I linked every step with the Issue, so that the changes can be undone without forgetting anything